dss-api.yml

```diff
@@ -345,6 +345,14 @@ paths:
                 pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
             required:
               - uuid
+        500:
+          description: Internal Server Error
+          schema:
+            $ref: '#/definitions/Error'
```

Comment:
You should look at GET /files to see how you should pattern this.  It should include the allowed error codes.